package.json

```diff
@@ -97,7 +97,6 @@
     "react-bootstrap": "0.32.4",
     "react-copy-to-clipboard": "5.0.1",
     "react-dom": "16.9.0",
-    "react-draggable": "3.2.1",
```

Comment:
We don't need `react-draggable` now?

Reply:
Correct, it didn't play well with PF4, even after an update to the most recent version of react-draggable.  So, I moved to a Popover-based approach.